src/interfaces/tokens/IProtocolToken.sol

```diff
@@ -8,7 +8,7 @@ import {
 } from '@openzeppelin-upgradeable/token/ERC20/extensions/ERC20VotesUpgradeable.sol';
 import {IAuthorizable} from '@interfaces/utils/IAuthorizable.sol';
 
-interface IProtocolToken is IVotesUpgradeable, IERC20MetadataUpgradeable, IERC20PermitUpgradeable, IAuthorizable {
```

Comment:
should remove?

Reply:
yes, we need to solve `Linearization of inheritance graph impossible`, the ProtocolToken is force to implemented  via `ERC20VotesUpgradeable` that imports `IERC5805Upgradeable` that is declared as `interface IERC5805Upgradeable is IERC6372Upgradeable, IVotesUpgradeable {}`

Reply:
Cool just wanted to be sure we aren't actually losing IVotesUpgradeable